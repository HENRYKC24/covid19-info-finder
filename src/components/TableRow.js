import React from 'react';

const TableRow = (props) => {
  const { param, value } = props;
  return (
    <tr className="table-row">
      <td>{param}</td>
      <td>{value.toLocaleString()}</td>
    </tr>
  );
};

export default TableRow;
