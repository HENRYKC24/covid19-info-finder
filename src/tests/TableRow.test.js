import React from 'react';
import renderer from 'react-test-renderer';
import TableRow from '../components/TableRow';

it('Navbar renders the UI as expected', () => {
  const tree = renderer.create(<TableRow param="Deaths Today" value={333} />).toJSON();
  expect(tree).toMatchSnapshot();
});
