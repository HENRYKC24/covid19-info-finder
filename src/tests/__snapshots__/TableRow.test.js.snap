// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders the UI as expected 1`] = `
<tr
  className="table-row"
>
  <td>
    Deaths Today
  </td>
  <td>
    333
  </td>
</tr>
`;
